### PR TITLE
[ticket/14654] Change Imagemagick to ImageMagick

### PR DESCRIPTION
**Pull request by Senky**
[PHPBB3-14654](https://tracker.phpbb.com/browse/PHPBB3-14654)

### CODE REVIEW

File: phpBB/language/en/acp/attachments.php [hunk 1]
```diff
@@ -117,7 +117,7 @@
 
 	'IMAGE_LINK_SIZE'			=> 'Image link dimensions',
 	'IMAGE_LINK_SIZE_EXPLAIN'	=> 'Display image attachment as an inline text link if image is larger than this. To disable this behaviour, set the values to 0px by 0px.',
-	'IMAGICK_PATH'				=> 'Imagemagick path',
+	'IMAGICK_PATH'				=> 'ImageMagick path',
 	'IMAGICK_PATH_EXPLAIN'		=> 'Full path to the imagemagick convert application, e.g. <samp>/usr/bin/</samp>.',
```
Comment (marc1706) [inline phpBB/language/en/acp/attachments.php]: This still says imagemagick. I guess that's based on the name of the executable?

Comment (Senky) [inline phpBB/language/en/acp/attachments.php]: Yes, you can tell by the first letter, which is also lowercase.

**Comment by marc1706** [comment]
I found a few more comments in files that still say "Imagemagick". I see you changed one comment to say "ImageMagick". Either change the others, too, or remove this change to the comment. BTW, it's causing a merge conflict due to the new installer in 3.2.

**Comment by Senky** [comment]
I have added one more change. But I don't understand why you mention merge conflict with 3.2, while this PR is against 3.1.

**Pull request merged**